contracts/ethereum/test/mmr.test.ts

```diff
@@ -0,0 +1,317 @@
+import { format } from "path";
+
+const { expect } = require("chai");
+const { ethers } = require("hardhat");
+
+const { blake2b } = require('@noble/hashes/blake2b');
+import * as s from "parity-scale-codec";
+
+
+
+describe("MMR contract", function () {
+  let mmr: any;
+  let owner: any;
+  beforeEach(async () => {
+    [owner] = await ethers.getSigners();
+    const MMRWrapper = await ethers.getContractFactory("MMRWrapper");
+    mmr = await MMRWrapper.deploy();
+  });
```

Comment:
Recommend using fixtures rather than `beforeEach` to increase test suit performance